perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -117,6 +137,11 @@
 # A global variable initialized once for Azure Blob Service
 _AZURE_BLOB_SERVICE = None
 
+BYTES_PER_KILOBYTE = 1024
+
+# If we would sleep less than 0.1 seconds, don't bother.
```

Comment:
why introduce this variable? If there is no clear "badness" of sleeping for less than 100ms, then we shouldn't need to have this.